gui/default/syncthing/device/shareDeviceIDDialogView.html

```diff
@@ -0,0 +1,39 @@
+<modal id="share-device-id-dialog" status="warning" icon="{{shareDeviceIdParams.icon}}" heading="{{shareDeviceIdParams.heading}}" large="{{ shareDeviceIdParams.method == 'email' ? 'yes' : 'no' }}" closeable="yes">
+  <div class="modal-body" ng-switch="shareDeviceIdParams.method">
+    <p>
+      <span translate>The following text will automatically be inserted into a new message.</span>
+      <span>
+        <span ng-switch-when="email">
+          <span translate>Your email app should open to let you choose the recipient and send it from your own address.</span>
+        </span>
+        <span ng-switch-when="sms">
+          <span translate>Your SMS app should open to let you choose the recipient and send it from your own number.</span>
+        </span>
+      </span>
+      <span translate>You can also copy and paste the content into the message manually.</span>
```

Comment:
I actually used "content" on purpose, so that we don't repeat the word "text" over and over. Do you think it's better to use "text" here as well?

Reply:
I found it a bit inconsistent while looking at the dialog, and thought it might be more confusing to talk about "content" here for the first time.  Better stick to the wording used right above ("The following text ...")

And while we're at it, I guess "paste the text into _a_ message manually" would fit better. When done manually, it's not really the same message mentioned before which supposedly opens automatically.

Reply:
Done.